### DISCUSSION TITLE
Returning after following a link (svelte)

### DISCUSSION THREAD

**Discussion by tataDan** [wailsapp/wails#1607] [Q&A]
Suppose I have this code in my App.svelte file:

` <p>Visit the <a href="https://svelte.dev/tutorial">Svelte tutorial</a> to learn how to build Svelte apps.</p>`

If run the application and click on the link, it does take me to the link.  However, I can not return to the application screen. If I click on the close button on the window that is being displayed (the web site), then the application closes. ` <Alt><Tab>` does not work either.

Any suggestions as to how to resolve this issue would be appreciated.

**Comment by leaanthony** [comment]
I think `window.history` might help here
